### ISSUE TITLE
Update to LIS3DH ADC init block

### ISSUE THREAD

**Issue by zfi** [parallaxinc/solo#311]
The ADC initialization now requires four values instead of one. The calibration process now expects to receive two pairs of values; zero set expected (ground or 0) and ground as measured; 3.3. volt expected and 3.3 volts as measured. I'm sure there are more technical terms for this, but this will do for now.

```
 * @param mV_L Value for ADC mV calibration - voltage at which lower calibration reading taken, in millivolts
 * @param mV_H Value for ADC mV calibration - voltage at which upper calibration reading taken, in millivolts
 * @param value_L Value for ADC mV calibration - value measured at lower calibration voltage, in millivolts
 * @param value_H Value for ADC mV calibration - value measured at upper calibration voltage, in millivolts
 *
 */
void lis3dh_adcCal_mV(lis3dh_t *device, int mV_L, int mV_H, int value_L, int value_H);

```

Update the ADC init configuration to capture values for the measured ground and 3.3 volt supply and generate a call to  lis3dh_adc_mV().

**Comment by zfi** [comment]
First pass of the new block looks like this:
![Screen Shot 2020-02-27 at 3 41 23 PM](https://user-images.githubusercontent.com/8442260/75497935-a3f75900-597a-11ea-9beb-2117f3b695b9.png)
Working on the prompt text. Any suggestions are welcome.

**Comment by MatzElectronics** [comment]
I'd recommend `Map ADC offset (mV) from ___ at 0V to ___ at 3.3V`

**Comment by zfi** [comment]
Verified working in SoloCup.

**Comment by zfi** [comment]
Released in Solo 1.4.2
Closing.
